src/main/webapp/style.css

```diff
@@ -0,0 +1,236 @@
+body {
+  background-color: #ececec;
+  color: black;
+  font-family: Aksara Bali Galang;
+}
+
+main {
+  margin: auto;
+  max-width: 1035px;
+  width: 100%;  
+}
+
+#content {
```

Comment:
Is this layer and these style necessary? It looks to me like this will essentially do the same thing as if it weren't there, but correct me if I'm missing anything.